neo/Cryptography/ECC/ECDsa.cs

```diff
@@ -104,5 +106,51 @@ public bool VerifySignature(byte[] message, BigInteger r, BigInteger s)
             BigInteger v = point.X.Value.Mod(curve.N);
             return v.Equals(r);
         }
+
+        public static ECPoint KeyRecover(ECCurve curve, BigInteger r, BigInteger s, byte[] msg, bool isEven, bool doChecks)
+        {
```

Comment:
We should validate that `r` and `s` are valid values in order to return an infinite point.

https://github.com/ethereum/go-ethereum/blob/fe9ffa5953a28d570478ac61238218a195fe0ccb/crypto/crypto.go#L239-L250

Reply:
We have convert "r" and "s" to valid values in system call part like what has done in "CalculateE"
But validate in funcation is good idea, we will move the convert part into the function